src/core/ui.js

```diff
@@ -47,7 +47,9 @@ function ariaDecorate(elem, ariaMap) {
 
 const respecUI = hyperHTML`<div id='respec-ui' class='removeOnSave' hidden></div>`;
 const menu = hyperHTML`<ul id=respec-menu role=menu aria-labelledby='respec-pill' hidden></ul>`;
+const closeButton = hyperHTML`<button class="close-button" aria-label="Close">❌</button>`;
```

Comment:
```suggestion
const closeButton = hyperHTML`<button class="close-button" title="Close">❌</button>`;
```

`title` also allows visual readers to see the button role by mouse hovering, while screen readers will still perceive it as `aria-label`.

Reply:
That's even better!